### PR TITLE
DOCSP 25032 Delete Diagrams

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Delete diagrams task page

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/docs-relational-migrator/DOCSP-25032/diagrams/manage-diagrams/delete-diagrams/

## JIRA

https://jira.mongodb.org/browse/DOCSP-25032

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65a03005dcc0144a515685a8

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 1]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
 
-Delete diagrams.
\ No newline at end of file
+You can delete diagrams on the :guilabel:`Mapping` tab of Relational
+Migrator. 
+
+About this Task 
+---------------
+
+- You must have at least two diagrams before you can delete a diagram
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:19]: [suggestion] I'd reverse the order here for clarity. Also, there's no need for a single item to be in an unordered list. 

"Relational Migrator requires at least one diagram in your project. You must have at least two diagrams before you can delete one diagram."

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 2]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to delete
+
+      Click the name of the diagram in the ribbon tab to display the
+      diagram. 
+
+   .. step:: Open the :guilabel:`Diagram Properties` pane
+
+      Click the :icon-fa5:`chevron-circle-right` arrow button on the
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:35]:
```suggestion
      Click the :icon-fa5:`chevron-circle-right` arrow button
```

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 3]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+      right of your window to open the :guilabel:`Diagram
+      Properties` pane. 
+
+      .. image:: /img/diagrams/diagram_properties.png
+         :alt: Diagram Properties pane
+
+      The :guilabel:`Diagram Properties` pane displays the active diagram name and
+      notes. 
+
+   .. step:: Click the :icon-fa5:`ellipsis-h` ellipsis button in the :guilabel:`Diagram Properties` pane
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:45]:
```suggestion
   .. step:: Click the :icon-fa5:`ellipsis-h` button in the :guilabel:`Diagram Properties` pane
```

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 4]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+
+      This opens a menu of diagram options. 
+
+      .. image:: /img/diagrams/diagram-options.png
+         :alt: Diagram Properties Options menu
+
+   .. step:: Click the :guilabel:`Delete Diagram` option
+
+      This opens a confirmation box.
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:54]:
```suggestion
      A confirmation box opens.  
```

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 5]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+         :alt: Diagram Properties Options menu
+
+   .. step:: Click the :guilabel:`Delete Diagram` option
+
+      This opens a confirmation box.  
+
+   .. step:: Click the :guilabel:`Delete Diagram` button in the confirmation box. 
+
+      Once you confirm that you want to delete the selected diagram,
+      Relational Migrator deletes the diagram. This step can not be
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:59]:
```suggestion
      Relational Migrator deletes the diagram. This step cannot be
```

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 6]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+      undone. 
+
+
+
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:63]: I think there's too much space here.

**Review by mdb-ashley — CHANGES_REQUESTED** [review]
Looking great so far, Lauren! I added a few suggestions for your consideration. Let me know when this is ready for another look.

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 7]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+   .. step:: Navigate to the diagram you want to delete
+
+      Click the name of the diagram in the ribbon tab to display the
+      diagram. 
+
+   .. step:: Open the :guilabel:`Diagram Properties` pane
+
+      Click the :icon-fa5:`chevron-circle-right` arrow button on the
+      right of your window to open the :guilabel:`Diagram
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 8]
```diff
@@ -10,4 +10,63 @@ Delete Diagrams
...
+
+      .. image:: /img/diagrams/diagram_properties.png
+         :alt: Diagram Properties pane
+
+      The :guilabel:`Diagram Properties` pane displays the active diagram name and
+      notes. 
+
+   .. step:: Click the :icon-fa5:`ellipsis-h` ellipsis button in the :guilabel:`Diagram Properties` pane
+
+      This opens a menu of diagram options.
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 4 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 5 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 6 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Fixed!

**Comment by ltran-mdb2** [comment]
Thank you for your feedback @mdb-ashley ! This should be ready for another look when you get a chance!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 9]
```diff
@@ -1,13 +1,71 @@
...
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Delete diagrams.
\ No newline at end of file
+You can delete diagrams on the :guilabel:`Mapping` screen of Relational
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:13]: [suggestion] We're in the RM docs, so I don't think we have to specify this.

```suggestion
You can delete diagrams on the :guilabel:`Mapping` screen.
```

File: source/diagrams/manage-diagrams/delete-diagrams.txt [hunk 10]
```diff
@@ -1,13 +1,71 @@
...
+Migrator.
```
Comment (mdb-ashley) [inline source/diagrams/manage-diagrams/delete-diagrams.txt:14]:
```suggestion

```

**Review by mdb-ashley — APPROVED** [review]
LGTM % one small suggestion. Thanks, Lauren!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 9 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

File: source/diagrams/manage-diagrams/delete-diagrams.txt (hunk 10 above)
Comment (ltran-mdb2) [reply source/diagrams/manage-diagrams/delete-diagrams.txt]: Done!

**Review by satyasinha — APPROVED** [review]

**Pull request merged**